ebos/equil/initstateequil_impl.hh

```diff
@@ -1278,6 +1279,15 @@ equilnum(const EclipseState& eclipseState,
         const auto& e = eclipseState.fieldProps().get_int("EQLNUM");
         std::transform(e.begin(), e.end(), eqlnum.begin(), [](int n){ return n - 1;});
     }
+    OPM_BEGIN_PARALLEL_TRY_CATCH();
+    const auto num_regions = eclipseState.getTableManager().getEqldims().getNumEquilRegions();
```

Comment:
Would you mind `static_cast<>`-ing `num_regions` into an `int`, please?  That way we don't get any additional [signed vs. unsigned comparison](https://ci.opm-project.org/job/opm-simulators-PR-builder/5040/gcc/folder.1194549214/source.f8f54d44-4296-4b6f-93aa-9161ed4b2ff4/#1284) diagnostics.